source/content/migrate-manual.md

```diff
@@ -159,7 +159,22 @@ Your **code** is all custom and contributed modules or plugins, themes, and libr
    - `vendor`
    - `sites`, excluding `sites/default/files`.
 
-  Refer to the "Base-Level Directories" section of [Drupal 8 Directory Structure](https://www.drupal.org/docs/8/understanding-drupal-8/directory-structure) for more details.
+  Refer to the "Base-Level Directories" section of [Drupal Directory Structure](https://www.drupal.org/docs/understanding-drupal/directory-structure) for more details.
+
+  </Tab>
+
+  <Tab title="Drupal 9" id="d9-code">
+
+  Copy the following directories from your existing site to a matching directory in your new site's `code/` directory:
```

Comment:
These instructions are not necessarily clear for Composer-managed sites. Some of the files in these locations might be committed to the repository (especially in `sites`), but others will be placed by Composer, and these should not be committed to the repository on Pantheon. To make matters more confusing, there is no accepted standard in the Drupal community whether Composer dependencies should be committed to the repository, or left out. The later is more common, but some users might have everything already committed.

It might be easiest in the Drupal 9 case to start with updating the .gitignore, and then have them commit everything that is not ignored.

Reply:
when you say updating .gitignore.. do you mean to add new lines for each non-custom package they're requiring via composer.json?

i.e. add to .gitignore, for example:
web/modules/ctools
web/modules/webform
etc?